### PR TITLE
update

### PR DESCRIPTION
**Pull request by maryhbb**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,2 +1 @@
-<app-star *ngFor= "let rating of ratings" [rating]=rating (starClicked)="onStarClicked($event)"></app-star>
-
+<app-star *ngFor="let rating of ratings" [rating]=rating (starClicked)="onStarClicked($event)"></app-star>
```
Comment (jdiemke) [inline src/app/app.component.html]: i think parameters like raitng have to be in quotation marks?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
 })
 export class AppComponent {
-  title = 'inputoutput';
+  title = 'input-output';
   ratings: number[] = [3.5, 4, 1.5, 2, 4.5, 2.5];
 
-
-  public onStarClicked(rating: number){
+  public onStarClicked(rating: number) {
     console.log(`The Rating ${rating} is clicked`)
```
Comment (jdiemke) [inline src/app/app.component.ts]: missing semicolon

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,20 +1,16 @@
...
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'inputoutput';
+  title = 'input-output';
   ratings: number[] = [3.5, 4, 1.5, 2, 4.5, 2.5];
```
Comment (jdiemke) [inline src/app/app.component.ts]: public or private?

**Review by jdiemke — CHANGES_REQUESTED** [review]
Looks great!

File: src/app/app.component.html (hunk 1 above)
Comment (maryhbb) [reply src/app/app.component.html]: yes, i corrected it.

File: src/app/app.component.ts (hunk 2 above)
Comment (maryhbb) [reply src/app/app.component.ts]: done.

File: src/app/app.component.ts (hunk 3 above)
Comment (maryhbb) [reply src/app/app.component.ts]: i did use public, is it right?

**Pull request merged**